ajax/libs/amcharts/package.json

```diff
@@ -1,14 +1,46 @@
 {
   "name": "amcharts",
   "filename": "amcharts.js",
-  "version": "3.13.0",
+  "version": "3.17.3",
   "description": "amCharts is a robust charting tool that will suit any dataviz need.",
   "keywords": [
-    "chart"
+    "chart",
+    "amcharts",
+    "javascript"
   ],
+  "author": "amCharts <contact@amcharts.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/amcharts/amcharts3.git"
+  },
   "homepage": "http://www.amcharts.com/",
   "license": {
     "name": "Free and commercial",
     "url": "http://www.amcharts.com/download/"
-  }
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/amcharts/amcharts3.git",
+    "basePath": "amcharts",
+    "files": [
+      "images/*.+(png|gif|svg)",
+      "lang/*.js",
+      "patterns/**/*.+(png|jpg)",
+      "plugins/dataloader/dataloader*.js",
+      "plugins/export/export*.+(css|js)",
+      "plugins/responsive/responsive*.+(js|map)",
+      "themes/*.js",
+      "amcharts*.js",
+      "funnel*.js",
+      "gantt*.js",
+      "gauge*.js",
+      "pie*.js",
+      "radar*.js",
+      "serial*.js",
+      "xy*.js"
```

Comment:
how about `**/*!(examples/**/*|libs/**/*|license.txt|LICENSE|readme.md|README.md)*
i'm not sure if these folders under plugins are needed:
https://github.com/amcharts/amcharts3/tree/master/amcharts/plugins/export/shapes
https://github.com/amcharts/amcharts3/tree/master/amcharts/plugins/export/lang